Control/public/common/utils.js

```diff
@@ -86,13 +87,20 @@ const getTaskShortName = (taskName) => {
 const parseOdcStatusPerEnv = (environment) => {
   try {
     if (environment.integratedServicesData && environment.integratedServicesData['odc']) {
-      const odc = JSON.parse(environment.integratedServicesData['odc']);
-      return odc.State;
+      const {State: state} = JSON.parse(environment.integratedServicesData['odc']);
+      const styleClass = state === 'RUNNING'
+        ? 'success'
+        : (state === 'READY'
+          ? 'primary'
+          : (state === 'ERROR'
+            ? 'danger'
+            : ''));
+      return {state, styleClass};
```

Comment:
Nested ternary are often hard to read, I suggest to use an object or a map

Reply:
Updated to use STATE_COLORS similar to AliECS